config/routes.rb

```diff
@@ -142,6 +142,7 @@
   get 'notes/liked' => 'notes#liked'
   get 'notes/image/:id' => 'notes#image'
   get 'notes/delete/:id' => 'notes#delete'
+  post 'notes/preview' => 'notes#preview'
```

Comment:
Hmm, @RuthNjeri - i believe this is live now and i tried going to https://publiclab.org/notes/preview?title=Hello%20there&body=Testing%20this to see if it worked... I got redirected to https://publiclab.org/tag/preview -- i'm sure it's working properly but what's the best way to try it out from a URL? Shouldn't GET params work the same as POST params? Thanks!

Reply:
Hey @jywarren ,
Hitting from browser directly will not work as it is GET, we can define a seperate GET route to handle this case.
For now, we have to use a curl request, I mentioned the template above in this thread.
Also this POST route is authenticated and we need to pass csrf token with the curl.

Reply:
Hi @jywarren, @Sagarpreet is right, the route only accepts POST requests, I have made an edit via this [PR](https://github.com/publiclab/plots2/pull/8755) to make it accept both POST and GET, please have a look

Reply:
Aha, that's awesome, thank you!!! And I see it already works at https://stable.publiclab.org/notes/preview?title=Hello%20there&body=Testing%20this -- that's great!!! Fantastic!!!